### PR TITLE
an example of a map custom section, using leaflet

### PR DESCRIPTION
**Pull request by paulfitz**
This creates a bare-bones map section, like in the `Hack Spots` example document.  Exercises `listTables` and `fetchTable`.

### CODE REVIEW

File: examples/example-6-map/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: map_section
+version: 0.0.1
+components:
+  safeBrowser: main.js
```
Comment (cpind) [inline examples/example-6-map/manifest.yml:4]: You should not have to declare a `safeBrowser` component.

File: examples/example-6-map/page.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+/* global grist, window */
+
+grist.ready();
+
+window.onload = function() {
+  const api = grist.rpc.getStub("GristDocAPI@grist", grist.checkers.GristDocAPI);
+  api.listTables()
+    .then(tables => {
+      return api.fetchTable(tables[0]);
```
Comment (cpind) [inline examples/example-6-map/page.js:11]: In a more elaborate plugin I would expect user to be able to select the table to read data from. But for an example I think picking the first one help keep the code concise and helps focuses on what matters.

File: examples/example-6-map/manifest.yml (hunk 1 above)
Comment (paulfitz) [reply examples/example-6-map/manifest.yml:4]: If I remove it, I currently get `Plugin <strong>installed/example-6-map</strong> was not found`.

File: examples/example-6-map/page.js (hunk 2 above)
Comment (paulfitz) [reply examples/example-6-map/page.js:11]: Likewise the names of the columns used for latitude and longitude would be made configurable or discovered.

File: examples/example-6-map/manifest.yml (hunk 1 above)
Comment (cpind) [reply examples/example-6-map/manifest.yml:4]: Arf .. this is a bug ;-( 
I'll look into it ... in the meantime I think we should accept this PR and update it later.

**Review by cpind — APPROVED** [review]
Looks good thank you.

**Pull request merged**